charts/opensearch-dashboards/values.yaml

```diff
@@ -155,3 +155,16 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+lifecycle: {}
+  # preStop:
+    # exec:
+  #     command: ["/bin/sh", "-c", "echo Hello from the postStart handler > /usr/share/message"]
+  # postStart:
+  #   exec:
+  #     command:
+  #       - bash
+  #       - -c
+  #       - |
+  #         #!/bin/bash
+  #       curl -X POST "localhost:5601/api/saved_objects/_import?createNewCopies=true -H "kbn-xsrf: true" --form file=@file.ndjson" -H 'kbn-xsrf: true'
```

Comment:
Is this the best way to determine ?

Reply:
I had just put it there as an example. If a working test is prefered. I can easily change it. @TheAlgo

Reply:
Yes that will be great. Thanks @bluehawk27

Reply:
Any progress @bluehawk27 ?

Reply:
@peterzhuamazon I'll knock this out tomorrow. I got swamped at work today.